# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: netwerk/protocol/http/PHttpChannel.ipdl
# Commit: effee4ab270c
# Full Hash: effee4ab270cba85ad53b7be844212bc458ec1a6
# Author: Honza Bambas <honzab.moz@firemni.cz>
# Date: 2019-10-03 04:38:57
# Regressor Bug: 1584207
# File Overlap Count: 1
# Description:
#   Bug 1584207 - Forward nsILoadInfo.requestBlockingReason to the parent process for both old and new redirect channels, r=dragana,asuth
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D47264
# ==============================================================================

diff -r a49132352f5b -r effee4ab270c netwerk/protocol/http/PHttpChannel.ipdl
--- a/netwerk/protocol/http/PHttpChannel.ipdl	Wed Oct 02 20:53:50 2019 +0200
+++ b/netwerk/protocol/http/PHttpChannel.ipdl	Tue Oct 01 16:50:23 2019 +0000
@@ -44,7 +44,8 @@
 
   // Reports approval/veto of redirect by child process redirect observers
   async Redirect2Verify(nsresult result, RequestHeaderTuples changedHeaders,
-                        ChildLoadInfoForwarderArgs loadInfoForwarder,
+                        uint32_t sourceRequestBlockingReason,
+                        ChildLoadInfoForwarderArgs targetLoadInfoForwarder,
                         uint32_t loadFlags, nsIReferrerInfo referrerInfo,
                         URIParams? apiRedirectTo,
                         CorsPreflightArgs? corsPreflightArgs,
